defense-unicorns-distro/zarf.yaml

```
@@ -54,6 +54,11 @@ variables:
   - name: EPHEMERAL
     description: "Set whether the cluster should be considered ephemeral - if true all resources will be force destroyed on removal"
     default: "false"
+  - name: DUBBD_VALUES
+    default: deploy-dubbd-values.yaml
```

I don't have a great suggestion here but I wonder if we can find a default that won't throw a warning on deploy if it's not provided 🤔 

```
 WARNING  Refusing to load a non-text file for templating ###ZARF_VAR_DUBBD_VALUES###                                                                                                                                   
                                                                                                                                                                                                                        
 WARNING  Refusing to load a non-text file for templating ###ZARF_VAR_ADD_DUBBD_VALUES###                                                                                                                               
```

---

@mjnagel - good catch!
addressed by including a default file (like [this one](https://github.com/defenseunicorns/uds-package-dubbd/blob/dubbd-values/defense-unicorns-distro/deploy-dubbd-values.yaml)) in the repo for each dubbd package and no longer receive that warning